accProcess.py

```diff
@@ -352,12 +352,6 @@ def deleteIntermediateFiles():
         fourierWithAcc=args.fourierWithAcc, m10l5=args.m10l5,
         verbose=args.verbose)
 
-    # Generate time series file (note: this will also resample to epochData so do this last)
-    accelerometer.accUtils.generateTimeSeries(epochData, args.tsFile,
-        epochPeriod=args.epochPeriod,
-        timeSeriesDateColumn=args.timeSeriesDateColumn,
-        activityClassification=args.activityClassification, labels=labels)
-
```

Comment:
This part has now been moved into summariseEpoch.py

Reply:
Is there a reason to move the -timeSeries.csv.gz writing code to within the summariseEpoch.py file?

To me this more logically fits as a utility function rather than something necessary to derive summary measures from the input epoch file.

Reply:
Thanks Aiden for taking the time to review the changes!

> 1. Can you move the Java timeZone updates to the EpochWriter.Java file?
>    This then means that this code is just needed one time, and can be used for .gt3x, .bin, and .csv input files too. This would certainly be helpful for the .gt3x and .bin files we process as these also store localDateTime.

I was thinking maybe it is preferable to perform the correction at the source instead of delegating this to subsequent processes and keep EpochWriter just in charge of writing/formatting. But I haven't considered .gt3x and .bin files and I'm not too familiar with them. Can you help verify that this is correct https://github.com/actigraph/GT3X-File-Format ? If the device's internal representation of time is in Unix time then it won't need DST correction.
Generally, I think because devices have their own format it makes semantic sense to have _their_ side of the parsing code handle what's necessary to output the correct values. What do you think?

> 2. Will this work for a scenario where the device is setup in one timezone, but deployed in another? i.e. should we keep the timeZoneOffset integer in our code?
>    @xiangnandang - perhaps you might like to comment on this?

Ah I see, in that case I will recover this functionality. But perhaps we should change its name to something like `timeShift` or `timeOffset`? Because timezone offset would be a bit misleading: If a device was configured in NY (UTC-5) and shipped to UK summertime (UTC+1) then the device needs +6 hours. `--timeZoneOffset 6` I think would be confusing.

> 3. The -timeSeries.csv.gz file intentionally did not write out the datetime column to save disk space. However, I imagine your reason for now including this column is to explicitly handle files which happen during a timeZone change?

Yes, because there can be DST crossovers and the offsets would change, e.g. if we're in Italy then the column might go `2020-03-29`**`01:59:00 +0100`**, `2020-03-29`**`01:59:30+0100`**, `2020-03-29`**`03:00:00+0200`**, ...

> 4. Can you add/keep some code to retain the 'quality-daylightSavingsCrossover' flag in the -summary.json file ?

Yes I will submit a fix soon. Thanks for spotting this.

> 5. Is there a reason to move the -timeSeries.csv.gz writing code to within the summariseEpoch.py file?
>    To me this more logically fits as a utility function rather than something necessary to derive summary measures from the input epoch file.

That makes sense, I will go back to the original behavior.